### ISSUE TITLE
Keywords to match trials on 

### ISSUE THREAD

**Issue by thejoefriel** [yalla-coop/curenetics#8]
Hi @sesola and @jbrough this is related to #2 but I thought it important enough to have as a separate issue for us to have a clear place to discuss. 

If we can't use Jamie's API to filter on eligibility criteria as such (beyond age, cancer type etc), then it might be possible to do some filtering based on keywords?

However, for this to be possible we would need to be 100% clear on:
- what keywords are listed for clinical trials
- what matching keywords might also be found in medical records
- what keywords would clearly indicate some form of eligibility if we find it in both the clinical trials and the medical records

For example, what we don't want is a clinical trial having a certain keyword that is actually only there because it means people wouldn't be eligible if they have that in their medical records, so we need to be clear what and why keywords are listed for clinical trials and whether these are always positive things we can match against. 

This may require the two of you discussing this week? And then we can add the thoughts as comments below.

**Comment by sesola** [comment]
Common clinical trials match criteria.  The component of the list may vary from trial to trial and  not an exhaustive list. I'm happy to chat further.

Inclusion criteria 
- Age 18 or over
- Eastern Cooperative Oncology Group (ECOG) status 0-2
- Diagnosis of  prostate cancer
- Prostate cancer on biopsy from a mpMRI defined lesion
- Normal full blood count and normal renal and liver function tests
- Any prior systemic therapy for prostate cancer( e.g. surgery, chemo, radiation)
- Co-morbidities( past medical problems)
-Stage of the cancer 
-Gleason grade on biopsy 


Exclusion criteria 
- To unwell, ECOG 3-4
- Lack of mental capacity 
-History of HIV or hepatitis B or C
-History of auto-immune disease 
-Prior treatment with immunotherapy
-Previous history of other types of cancer 
-Active invasive malignancy in the previous 2 years excluding non-melanoma skin cancer.
-Significant cardiovascular disease, myocardial infarction within 3 months prior to enrolment, 
-History of QT prolongation
-Men unable to have MRI scan, or in whom artefact would significantly reduce quality of MR
-Contraindications to biopsy
-Unable to communicate in English

**Comment by sesola** [comment]
sorry I mixed up my bullet point signs.

**Comment by thejoefriel** [comment]
Hi @sesola - sorry I think we still need this to be made much simpler for us (at least me) to understand! 

For each of these - what words could we expect to see on the medical record, which Jon will pull out AND what sort of relevant keywords would we look for in the list of clinical trials?

For example, diagnosis of prostate cancer - what are the full list of terms that might be provided? 

I've provided a spreadsheet template that might be useful for you and maybe with help from @jbrough to fill in? https://docs.google.com/spreadsheets/d/1vwfUVFS4Rov0nnbV-o2ew6kcAnMzbXXmI3xgfGluEME/edit?usp=sharing

Basically, we have no expertise in this domain, so if we are going to have a chance to match on anything other than age, gender, type of cancer, we need a super clear list for each potential criteria, what would be the keywords we could pull out of a medical record that might be listed in a clinical trial which we can 100% assume will mean the patient either meets or doesn't meet that particular criteria. 

Hope that makes sense! Let me know if you'd like to chat through

**Comment by thejoefriel** [comment]
This has been agreed to be ECOG status, gleason score and disease outside / within prostate
so closing this issue now
